docs/MVS_Outputs.rst

```diff
@@ -282,8 +282,11 @@ As above, we apply a weighting based on Electricity Equivalent.
 Degree of net zero energy (NZE)
 ###############################
 
-The degree of net zero energy describes the ability of an energy system to provide its aggregated annual demand though renewable, local sources. For that, the balance between local generation as well as consumption from and feed-in towards the energy provider is compared. In a net zero energy system, demand can be supplied by energy import, but then local renewable energy generation should allow an equally high energy export of energy in the course of the year. In a plus energy system, the export exceeds the import, while local renewable generation can supply all demand (from an aggregated perspective).
-To calculate the degree of NZE, the overall renewable energy generated locally minus the excess energy is divided by the overall energy consumed.
+The degree of net zero energy describes the ability of an energy system to provide its aggregated annual demand though local sources. For that, the balance between local generation as well as consumption from and feed-in towards the energy provider is compared.
+In a net zero energy system, demand can be supplied by energy import, but then local energy generation should allow an equally high energy export of energy in the course of the year. In a plus energy system, the export exceeds the import, while local generation can supply all demand (from an aggregated perspective).
+To calculate the degree of NZE, the overall grid feed-in minus the grid consumption is added with one and afterwards divided by the overall energy consumed.
```

Comment:
```suggestion
To calculate the degree of NZE, the margin between grid feed-in and grid consumption is compared to the overall demand. 
```
I think "is added with one" may raise some question marks. We can also say that the inverse of the margin of E_f and E_c devided by the demand is the degree of NZE, but that is also not very clear...